tandem/main.py

```diff
@@ -138,20 +138,26 @@ def execute(args,
     Raises:
         Input error if the number of RMG tolerances is greater than the total number of T3 iterations.
     """
-    rmg_input_file = args.file
+    cwd = os.getcwd()
+    # If the output directory is not yet set, then set it to the same directory as the input files by default
+    if not args.output_directory:
+        args.output_directory = cwd
```

Comment:
I didn't see where `args.output_directory` is actually being used. Does it actually work and relocates the entire working folder (all iterations inc. ARC runs)?

Reply:
Various lines in tandem/main.py use args.output_directory so I defined it here and added it to the parse_command_line_arguments() function in t3.py. Is this what you're asking about? Help me understand the question

Reply:
I see. Did you test the `-o` argument to see that it works smoothly?

Reply:
I had not, which was my bad. Using `-o` did not work as I had expected since args.output_directory is a list with one entry containing the directory as a string... trying to resolve now but I'm unsure why it doing `-o "<path>"` at the command prompt results in a list

Reply:
I needed to read the documentation more closely. The default `action` is to just store the argument's value and not create a list, but by using `nargs=1` or `nargs=n` that gathers the n arguments into a list. Problem solved. 

I'll address your comments below by putting the whitespace back and then squash. Thanks for your feedback!